scripts/run-python-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Test python model wrapping inside virtual environment
+
+AMICI_PATH="`dirname \"$0\"`"
+AMICI_PATH="`( cd \"$AMICI_PATH/..\" && pwd )`"
+
+cd ${AMICI_PATH}/tests
+
+${AMICI_PATH}/build/venv/bin/pip3 install scipy
```

Comment:
`source /build/venv/bin/activate`?

Reply:
There was an issue with `activate` that made me use above. I don't recall the details anymore.